ResourceStatusSystem/Command/PropagationCommand.py

```diff
@@ -0,0 +1,53 @@
+''' PropagationCommand module
+'''
+
+from DIRAC                                                      import S_OK, S_ERROR
+from DIRAC.ResourceStatusSystem.Client.ResourceStatusClient     import ResourceStatusClient
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.ResourceStatusSystem.Utilities                       import CSHelpers
+
+__RCSID__ = '$Id:  $'
+
+class PropagationCommand( Command ):
+
+  def __init__( self, args = None, clients = None ):
+
+    self.rssClient = ResourceStatusClient()
+    super( PropagationCommand, self ).__init__( args, clients )
+
+  def doNew( self, masterParams = None ):
+     return S_OK()
+
+  def doCache( self ):
+
+    if not self.args['site']:
+      return S_ERROR('site was not found in args')
+
+    site = self.args['site']
+
+    storageElements = CSHelpers.getSiteStorageElements(site)
+    computingElements = CSHelpers.getSiteComputingElements(site)
```

Comment:
I just created a generic `getSiteElements` as you suggested, however given the current structure in the CS:
```
    Sites
    {
        LCG
        {
            LCG.somesite.site
            {
                SE = element1.test.ch
                CE = computing1.test.ch
             ...
```

I'm afraid that I can not avoid hardcoding some things in the `getSiteElements` like calling `gConfig.getValue( '%s/%s/%s/SE' % ( _basePath, domainName, siteName ), '' )`, `gConfig.getValue( '%s/%s/%s/CE' % ( _basePath, domainName, siteName ), '' )`, etc and concatenate the results..